### DISCUSSION TITLE
paperless-ngx: consume folder?

### DISCUSSION THREAD

**Discussion by einsq** [tteck/Proxmox#1126] [Help Needed]
Where can i configure a consume folder for paperless-ngx? And how can i send via my scanner (Brother ADW-1700) direct to paperless-ngx? I can configure a mail input but no network input.

**Comment by Vakrehus** [comment]
SMB (install samba inside the paperless-lxc) works perfektly with my Brother ADS-1600W.

This is my `/etc/samba/smb.conf`
```
#======================= Share Definitions =======================

[global]
map to guest = never

[homes]
comment = Home Directories
browsable = no
read only = yes
valid users = %S
create mask = 0700
directory mask = 0700

[paperless]
comment = Paperless SMB Consume
valid users = root
path = /opt/paperless/consume/
public = no
writable = yes
printable = no
browsable = no
guest ok = no
create mask = 0700
directory mask = 0700
```

**Reply by malosaa** [reply]
Can you tell me what commands to run inside the console of paperless-lxc ?
I'm still learning proxmox, would love to have an smb share going for paperless 

// update 
i just run apt install SMB and used your config 
than i did run smbpasswd -a root to setup a password and it works now :) 

thanks
